helm/ffc-sfd-file-receiver-infra/values.yaml

```diff
@@ -1,3 +1,10 @@
 # For documentation on how to use, please go here: https://github.com/DEFRA/adp-aso-helm-library?tab=readme-ov-file#adp-platform-azure-service-operator-helm-library-chart.
 # You may create Queues, Topics, Subscriptions, Storage Accounts, Databases and Role Assignments on these here.
-
+namespace: fcp-sfd
+namespaceTopics:
+  - name: 'file-metadata'
+    roleAssignments:
+      - roleName: 'TopicSender'
+      - roleName: 'TopicReceiver'
+    topicSubscriptions:
+      - name: 'file-receiver'
```

Comment:
After adding ADP's Service Bus template (`namespace-topic.yaml`/`namspace-queue.yaml`) into the Infra Helm chart, you'll also need to update the `values.yaml` file (also located in the Infra Helm chart) to include something similar to the above. You can see that we have the namespace `fcp-sfd` declared at the top as the topic and subscription prefix. Underneath this we tell ADP's pipelines what the rest of the topic/subscription name should be on lines 5 and 10. Personally I assign topics/subscriptions both roles of `TopicSender` and `TopicRecevier` so that way if you ever need to configure a microservice to act as a `MessageSender`, then the capability is there and you're not limited.